File: Task/Rot-13/Clojure/rot-13-2.clj
(let [A (into #{} "abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ")
      A-map (zipmap A (take 52 (drop 26 (cycle A))))]

  (defn rot13[in-str]
    (reduce str (map #(if (A %1) (A-map %1) %1)  in-str))))

(rot13 "The Quick Brown Fox Jumped Over The Lazy Dog!") ;; produces "Gur Dhvpx Oebja Sbk Whzcrq Bire Gur Ynml Qbt!"

File: Task/Rot-13/Python/rot-13-2.py
>>> import codecs
>>> codecs.encode("The quick brown fox jumps over the lazy dog", "rot13")
'Gur dhvpx oebja sbk whzcf bire gur ynml qbt'
>>> codecs.decode(_, "rot13")
'The quick brown fox jumps over the lazy dog'
